scala-package/native/src/main/native/org_apache_mxnet_native_c_api.cc

```diff
@@ -1581,26 +1581,29 @@ JNIEXPORT jint JNICALL Java_org_apache_mxnet_LibInfo_mxSymbolInferShape
   env->ReleaseIntArrayElements(jargShapeData, argShapeData, 0);
   env->ReleaseIntArrayElements(jargIndPtr, argIndPtr, 0);
 
-  jclass listClass = env->FindClass("scala/collection/mutable/ListBuffer");
-  jmethodID listAppend = env->GetMethodID(listClass,
-    "$plus$eq", "(Ljava/lang/Object;)Lscala/collection/mutable/ListBuffer;");
+  if (ret == 0) {
+    jclass listClass = env->FindClass("scala/collection/mutable/ListBuffer");
+    jmethodID listAppend = env->GetMethodID(listClass,
+      "$plus$eq", "(Ljava/lang/Object;)Lscala/collection/mutable/ListBuffer;");
 
-  if (FillSymbolInferShape(env, listAppend, jinShapeData, inShapeSize, inShapeNdim, inShapeData)) {
-    // TODO(Yizhi): out of memory error thrown, return a specific error code ?
-    return -1;
-  }
-  if (FillSymbolInferShape(
-        env, listAppend, joutShapeData, outShapeSize, outShapeNdim, outShapeData)) {
-    // TODO(Yizhi): out of memory error thrown, return a specific error code ?
-    return -1;
-  }
-  if (FillSymbolInferShape(
-        env, listAppend, jauxShapeData, auxShapeSize, auxShapeNdim, auxShapeData)) {
-    // TODO(Yizhi): out of memory error thrown, return a specific error code ?
-    return -1;
-  }
+    if (FillSymbolInferShape(
```

Comment:
please don't change the code ident if you are not changing the content inside

Reply:
What @lanking520 is saying is that we follow the [style-guide](https://github.com/apache/incubator-mxnet/blob/master/scala-package/scalastyle-config.xml) and request not make whitespace changes.

Reply:
Hi! This change placed these three `if (FillSymbolInferShape...` calls inside a new block, the `if (ret == 0) {` now on line 1584. Are you saying indentation shouldn't be changed even if a new scope is introduced around the code?

As for inserting the line break on the first of those calls, without that, your own style checker failed because that line was > 100 characters. I followed the convention with the rest of this file to make that line shorter.

Am I missing something, or do any of these changes actually violate your style guide?